code/datalab-app/src/app/components/welcome/DescribeElement.js

```diff
@@ -16,12 +16,13 @@ const DescribeElement = ({ classes, title, descriptions, invert, quote, media })
   <DescribeElementSegment invert={invert}>
     <Typography className={classes.title} type="display1">{title}</Typography>
     <Grid container align="stretch">
-      {descriptions.map(({ icon, title: cardTitle, content: cardContent }, idx) => (
+      {descriptions.map(({ icon, title: cardTitle, content: cardContent, links }, idx) => (
```

Comment:
I don't understand this. It looks like you're destructuring and assigning the value for `cardTitle` to `title` but then using the variable `cardTitle` in the assignment below. It must work or it wouldn't render, I just don't understand how...

Reply:
Confusingly in ES6 it is the other way around, I am assigning the `title` value to `cardTitle`. The reason for the rename is the variable title is already used.